uber/decorators.py

```diff
@@ -500,28 +500,37 @@ def __call__(self, func):
         return func
 
 
-def attendee_id_required(func):
-    @wraps(func)
-    def check_id(*args, **params):
-        message = "No ID provided. Try using a different link or going back."
-        session = params['session']
+def id_required(model):
+    def model_id_required(func):
+        @wraps(func)
+        def check_id(*args, **params):
+            check_id_for_model(model=model, **params)
```

Comment:
btw, with this (awesome) change, you can probably refactor the contents of check_id_for_model back inline here.  either way is fine though